### ISSUE TITLE
2.6 Ceilings -- clarification

### ISSUE THREAD

**Issue by jswingle** [gryphonshafer/cbqz-site#1]
The following text should be clarified:

```
2.6. Ceilings
A quizzer remains eligible to trigger for queries until they accrue correct rulings on:

4 queries should none of the first 3 of those queries be open-book
3 queries should any 1 but only 1 of the first 2 queries be open-book
2 queries should all be open-book
A team is limited to 3 open-book queries per quiz.
```

Is this intended to be 3 total calls for open-book, or 3 _correct_? I would recommend "A team is limited to 3 _correct_ open-book queries per quiz."

**Comment by jswingle** [comment]
I think the context strongly points to assuming it as "correct" but it _is_ formatted outside of the other bullet points so someone could argue otherwise.

**Comment by gryphonshafer** [comment]
Agreed. Resolved with c184181.
